docker/Dockerfile

```
@@ -164,6 +164,7 @@ RUN --mount=type=cache,target=/root/.cache/uv \
 # see https://github.com/pytorch/pytorch/pull/123243
 ARG torch_cuda_arch_list='7.0 7.5 8.0 8.9 9.0 10.0 12.0'
```

```suggestion
ARG torch_cuda_arch_list='7.0 7.5 8.0 8.9 9.0 10.0 12.0+PTX'
```
Should allow it to attempt to run on newer architectures btw with JIT if it's able to.

---

IIRC we handle adding +PTX on a kernel-by-kernel basis within our cmake build. Either way, not going to touch it in this PR